draft-ietf-quic-transport.md

```diff
@@ -2709,8 +2710,10 @@ frame risks a peer missing the first such packet.  The only mechanism available
 to an endpoint that continues to receive data for a terminated connection is to
 use the stateless reset process ({{stateless-reset}}).
 
-An endpoint that receives an invalid CONNECTION_CLOSE frame MUST NOT signal the
-existence of the error to its peer.
+An endpoint that receives an invalid CONNECTION_CLOSE frame MUST treat the
```

Comment:
What is an invalid CONNECTION_CLOSE frame?

Reply:
One that has an reason phrase that overruns the packet would do it.  I can't see any other reason.

Reply:
I don't think we need any extra rules for this special case. Just do what you'd normally do: close the connection with a FRAME_ENCODING_ERROR.

Reply:
I see, you think that it is enough to rely on rate limiting of the CONNECTION_CLOSE packet (c.f., #3095).  That would also work.